www/docs/monitoring-your-app-in-prod.md

```diff
@@ -0,0 +1,162 @@
+---
+id: monitoring-your-app-in-prod
+title: Monitoring Your App in Prod
+sidebar_label: Monitoring Your App in Prod
+description: "How to use services like Datadog, Sentry, Epsagon, and Lumigo to monitor the Lambda functions in your SST app in production."
+---
+
+import TabItem from "@theme/TabItem";
+import MultiLanguageCode from "@site/src/components/MultiLanguageCode";
+
+Once your app has been [deployed to production](deploying-your-app.md), it's useful to be able to monitor your Lambda functions. There are a few different services that you can use for this. We'll look at them below.
+
+## Datadog
+
+Datadog offers an [End-to-end Serverless Monitoring](https://www.datadoghq.com/product/serverless-monitoring/) solution that works with Lambda functions. The best way to integrate is by using the [CDK construct](https://github.com/DataDog/datadog-cdk-constructs) they provide.
+
+Start by adding it to your project.
+
+```bash
+# npm
+npm install --save-dev datadog-cdk-constructs
+
+# Yarn
+yarn add --dev datadog-cdk-constructs
+```
+
+Next, you'll need to import it into a stack. Add pass in the functions you want monitored.
+
+```js
+import { Datadog } from "datadog-cdk-constructs";
+
+const datadog = new Datadog(this, "Datadog", {
+  apiKey: process.env.DATADOG_API_KEY
+})
+
+datadog.addLambdaFunctions([myfunc]);
+```
+
+To monitor all the functions in a stack, you can use the Stack construct's [`getAllFunctions`](constructs/Stack.md#getallfunctions) method and do the following.
+
+```js
+datadog.addLambdaFunctions(this.getAllFunctions());
```

Comment:
Once we add the `getAllFunctions` to the stack, double-check that this code snippet is correct.